assets/css/live_view.css

```diff
@@ -18,7 +18,3 @@ iframe[hidden] {
 .phx-loading {
   cursor: wait;
 }
-
-.phx-loading * {
-  pointer-events: none;
-}
```

Comment:
Does this mean we can try to click a button though? And then it will crash?

Reply:
This happens when the LV crashes and the socket disconnects, in which case clicking doesn't do anything.

Reply:
Also, the phoenix generator no longer adds this CSS, so that's another reason :)